JPS_BASE_LIB/python_wrapper/README.md

```diff
@@ -1,10 +1,10 @@
-# What is py4jps
+# What is `twa`
 
-`Py4jps` is a thin Python wrapper for the [TheWorldAvatar](https://github.com/cambridge-cares/TheWorldAvatar) project. The code is heavily based on the [py4j](https://www.py4j.org/index.html) package, which enables Python programs running in a Python interpreter to dynamically access Java objects in a Java Virtual Machine.
+`twa` is a thin Python wrapper for the [TheWorldAvatar](https://github.com/cambridge-cares/TheWorldAvatar) project. The code is heavily based on the [py4j](https://www.py4j.org/index.html) package, which enables Python programs running in a Python interpreter to dynamically access Java objects in a Java Virtual Machine. It has a precedent python package, `py4jps`, which is now deprecated.
```

Comment:
I'd remove the "thin" and the redundant "the" (in front of "TheWorldAvatar").

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1149/commits/01a8ee11d9e4caa7e77be0d9fdd9ffd51088d91a